packages/server/src/messages/api.ts

```diff
@@ -169,7 +167,7 @@ export class MessageApi extends ClientScopedApi {
         })
       }
 
-      const msg = await this.chat.send(conversationId, userId, payload, { socket })
+      const msg = await this.messages.create(conversationId, userId, payload) //, { socket })
```

Comment:
Chat api does not exist anymore. We'll have to add a `suppressEvents` options or something here to allow the channel web to only create messages without everything around.

Reply:
Turns out we actually don't need to do this